materialize-snowflake/sqlgen.go

```diff
@@ -266,6 +290,14 @@ WHEN NOT MATCHED THEN
 		r.{{$key.Identifier -}}
 	{{- end -}}
 );
+{{ end }}
+
+{{ define "copyHistory" }}
+SELECT FILE_NAME, STATUS, FIRST_ERROR_MESSAGE FROM TABLE(INFORMATION_SCHEMA.COPY_HISTORY(
+  TABLE_NAME=>'{{ $.Table.Identifier }}',
+  START_TIME=>DATEADD(MINUTE, -30, TO_TIMESTAMP_LTZ('{{ $.StartTime }}'))
```

Comment:
This makes sense, I switched to always searching the last 14 days, but I also checked the behaviour in case of asking for a `START_TIME` that's more than 14 days ago, and there is no error, just that no records older than 14 days will be present in the response.